tests/test_helper.go

```diff
@@ -0,0 +1,337 @@
+				"migration_log_reader":      true,
+				"olm_managed":               true,
+				"cluster_name":              getEnvStr(CLUSTER_NAME, "host"),
+				"restic_timeout":            getEnvStr(RESTIC_TIMEOUT, "1h"),
+				"migration_velero":          getEnvBool(MIGRATION_VELERO, true),
+				"rsync_transfer_image_fqin": getEnvStr(RSYNC_TRANSFER_IMAGE_FQIN, "quay.io/konveyor/rsync-transfer:latest"),
+				"mig_pv_limit":              getEnvInt(MIG_PV_LIMIT, 100),
+				"version":                   getEnvStr(VERSION, "latest"),
+				"azure_resource_group":      "",
+			},
+		},
+	}
+}
+
+func NewMigrationNS(ns string) *v1.Namespace {
+	return &v1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: ns,
+		},
+	}
+}
+
+func GetMigSaToken(sourceClient *kubernetes.Clientset) []byte {
+	ctx := context.TODO()
+	sa, err := sourceClient.CoreV1().ServiceAccounts(MigrationNamespace).Get(ctx, MigrationController, metav1.GetOptions{})
+	Expect(err).NotTo(HaveOccurred())
+	for _, s := range sa.Secrets {
+		if strings.Contains(s.Name, "token") {
+			secret, err := sourceClient.CoreV1().Secrets(MigrationNamespace).Get(ctx, s.Name, metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			return secret.Data["token"]
+		}
+	}
+	return nil
+}
+
+func getEnvInt(name string, val int) int {
+	t, err := strconv.Atoi(os.Getenv(name))
+	if err != nil {
+		return val
+	}
+	return t
+}
+
+func getEnvStr(name string, val string) string {
+	t := os.Getenv(name)
+	if t == "" {
+		return val
+	}
+	return t
+}
+
+func getEnvBool(name string, val bool) bool {
+	t, err := strconv.ParseBool(os.Getenv(name))
+	if err != nil {
+		return val
+	}
+	return t
+}
+
+func NewPVC(name string, namespace string) *v1.PersistentVolumeClaim {
+	storage, err := resource.ParseQuantity("2Gi")
+	Expect(err).ToNot(HaveOccurred())
+
+	return &v1.PersistentVolumeClaim{
+
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: v1.PersistentVolumeClaimSpec{
+			AccessModes: []v1.PersistentVolumeAccessMode{"ReadWriteOnce"},
+			Resources: v1.ResourceRequirements{
+				Requests: v1.ResourceList{
+					"storage": storage,
+				},
+			},
+		},
+	}
+}
+
+func NewDeploymentFor41583() *appsv1.Deployment {
+	res := int32(1)
+	return &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "longpvc-test",
+			Namespace: "ocp-41583-longpvcname",
+			Labels: map[string]string{
+				"app": "longpvc-test",
+			},
+		},
+		Spec: appsv1.DeploymentSpec{
+			Replicas: &res,
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					"app": "longpvc-test",
+				},
+			},
+			Template: v1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"app": "longpvc-test",
+					},
+				},
+				Spec: v1.PodSpec{
+					Containers: []v1.Container{
+						{
+							Name:            "pod-test",
+							ImagePullPolicy: "Always",
+							Image:           "quay.io/jgabani/dg:latest",
```

Comment:
This should be somewhere other than your personal org, probably in Konveyor auto-building from a Quay build job.

Reply:
Replaced with the image `pvc-benchmark-helper` from `konvoyer`